src/richie/apps/core/context_processors.py

```diff
@@ -32,6 +32,9 @@ def site_metas(request):
             "domain": site_current.domain,
             "web_url": f"{protocol:s}://{site_current.domain:s}",
         },
+        "RICHIE_APPS": {
```

Comment:
in commit message

`s/if an optional Richie app is installed our not/if an optional Richie app is installed
or not/`